specifications/loopback-4-filter-specifications.md

```diff
@@ -0,0 +1,270 @@
+# The LoopBack 4 Filter Specification - Draft 1
```

Comment:
Should we perhaps emphasize the fact that this is a work in progress? Maybe add a paragraph with content similar to what we have in experimental packages?

```md
## Status: ⚠️WORK IN PROGRESS⚠️

> The specification is not complete yet and some parts may be inaccurate.
```